test/common/test_common.cpp

```diff
@@ -284,6 +284,139 @@ TEST (PCL, PointCloud)
   EXPECT_FALSE (cloud.isOrganized ());
   EXPECT_EQ (cloud.width, 2);
   EXPECT_EQ (&new_point, &cloud.back ());
+
+  cloud.resize (640*360);
+  EXPECT_EQ (cloud.width, 640*360);
+  EXPECT_EQ (cloud.height, 1);
+
+  cloud.resize (640, 480);
+  EXPECT_EQ (cloud.width, 640);
+  EXPECT_EQ (cloud.height, 480);
+
+  cloud.resize (640*360, PointXYZ (1, 1, 1));
+  EXPECT_EQ (cloud.width, 640*360);
+  EXPECT_EQ (cloud.height, 1);
+
+  cloud.resize (640, 480, PointXYZ (1, 1, 1));
+  EXPECT_EQ (cloud.width, 640);
+  EXPECT_EQ (cloud.height, 480);
+
+  cloud.assign (640*360, PointXYZ (1, 1, 1));
+  EXPECT_EQ (cloud.width, 640*360);
+  EXPECT_EQ (cloud.height, 1);
+
+  cloud.assign(640, 480, PointXYZ (1, 1, 1));
+  EXPECT_EQ (cloud.width, 640);
+  EXPECT_EQ (cloud.height, 480);
+
+  std::vector<PointXYZ> pointVec;
+  pointVec.resize (640*360, PointXYZ (2, 3, 4));
+  cloud.assign (pointVec.begin(), pointVec.end());
+  EXPECT_EQ (cloud.width, 640*360);
+  EXPECT_EQ (cloud.height, 1);
+
+  pointVec.resize(640*480, PointXYZ (3, 4, 5));
+  cloud.assign (pointVec.begin(), pointVec.end(), 640);
+  EXPECT_EQ (cloud.width, 640);
+  EXPECT_EQ (cloud.height, 480);
+
+  pointVec.resize (640*480, PointXYZ (7, 7, 7));
+  cloud.assign (pointVec.begin(), pointVec.end(), 460);
+  EXPECT_EQ (cloud.width, 640*480);
+  EXPECT_EQ (cloud.height, 1);
+
+  //initializer_list
+  cloud.assign ({PointXYZ (3, 4, 5), PointXYZ (3, 4, 5), PointXYZ (3, 4, 5), PointXYZ (3, 4, 5), PointXYZ (3, 4, 5),
```

Comment:
Can we make it easier to read? Perhaps just 2 or 3 points (and same below, with 2-3 points per line, and 2-3 lines at max) would suffice to show the test.